core/declarations.gms

```diff
@@ -282,6 +282,7 @@ vm_dummyBudget(ttot,all_regi)                        "auxiliary variable that he
 ***----------------------------------------------------------------------------------------
 ***-------------------------------------------------ESM module-----------------------------
 vm_macBase(ttot,all_regi,all_enty)                   "baseline emissions for all emissions subject to MACCs (type emismac)"
+vm_emiCO2_sector(ttot,all_regi,emi_sectors)          "total CO2 emissions from individual sectors [GtC]"
```

Comment:
please try to avoid "_" and use capital letters instead to shorten variable and parameter names :-)

Reply:
I changed:
``` R
cm_NDC_divergent_scenario to cm_NDC_divergentScenario
cm_CO2_tax_sector_markup to cm_CO2TaxSectorMarkup
vm_emiCO2_sector to vm_emiCO2Sector
q_emiCO2_sector to q_emiCO2Sector
p21_extRegi_CO2_tax_sector_markup to p21_extRegiCO2TaxSectorMarkup
p21_CO2_tax_sector_markup to p21_CO2TaxSectorMarkup
p21_taxrevCO2_sector0 to p21_taxrevCO2Sector0
v21_taxrevCO2_sector to v21_taxrevCO2Sector
q21_taxrevCO2_sector to q21_taxrevCO2Sector
p21_taxrevCO2_sector0 to p21_taxrevCO2Sector0
```